src/utils/protocol-events-map.ts

```diff
@@ -74,19 +80,11 @@ export const protocolEventsMap: {readonly [key in ProtocolEvent]: ProtocolEventI
     validateAndGetSqsEvents: (interestingTransaction: InterestingTransaction) =>
       getSqsEventsFromTx(ProtocolEvent.BatchDeployment, interestingTransaction),
   },
-  [ProtocolEvent.LegacyMintNft]: {
-    name: ProtocolEvent.LegacyMintNft,
-    contractMethodName: ContractMethodId.cxipMint,
-    events: [eventMap[EventType.TransferERC721], eventMap[EventType.HolographableContractEvent]],
-    sqsEventNames: [SqsEventName.MintNft],
-    validateAndGetSqsEvents: (interestingTransaction: InterestingTransaction) =>
-      getSqsEventsFromTx(ProtocolEvent.LegacyMintNft, interestingTransaction),
-  },
   [ProtocolEvent.MoeMintNft]: {
     name: ProtocolEvent.MoeMintNft,
     contractMethodName: ContractMethodId.purchase,
     events: [
-      eventMap[EventType.TransferERC721],
+      // eventMap[EventType.TransferERC721],
```

Comment:
Why did you comment on these lines? Can you add some TODO's?

Reply:
I'm going to add a Note to this. 
The goal of this is to map all events that happen. But transfer checks are skipped for efficiency. Filtering transfer logs is costly, and all relevant data is already available in the HolographableContractEvent.